pkg/sdk/poc/generator/templates/struct.tmpl

```diff
@@ -1,5 +1,6 @@
 {{- /*gotype: github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/poc/generator.Field*/ -}}
 
+// 
```

Comment:
Adds spaces between structs

Reply:
How does it work? I'm guessing comments in templates are ignored, but wouldn't deleting the `//` result in the same output?

Reply:
wouldn't just adding a newline after the struct def we enough?

Reply:
why? can't we also have a new line after the "main" struct?

Reply:
yeah we can do that this way, there will be a newline after the struct in the `struct.tmpl` and a newline in `operation_struct.tmpl` after the struct

Reply:
we will do this by adding a trailing newline under structs for `struct.tmpl` and `operation_struct.tmpl` as proposed by @sfc-gh-asawicki above